### ISSUE TITLE
How to move bar and it's text to the right in Stacked BarChart ?

### ISSUE THREAD

**Issue by kaushikr-nimblechapps** [wuxudong/react-native-charts-wrapper#770]
Hi,

I have used the Stacked BarChart, Data binding is perfectly but I need to move the first bar with some padding. first bar is not display properly in the chart, half part hide in it, So please help me what's props for padding for it.

Image attached.

![Untitled](https://user-images.githubusercontent.com/28249592/100733880-d8f67780-33f4-11eb-8923-9f2ede522be3.png)

**Comment by aysenurodabasi** [comment]
Any solution on this?

**Comment by kaushikr-nimblechapps** [comment]
Yes just remove axisMinimum: 0, and axisMaximum: 5 from xAxis

**Comment by aysenurodabasi** [comment]
> Yes just remove axisMinimum: 0, and axisMaximum: 5 from xAxis.

Yes its okay but I have group datas. This settings breaks them :(
